hoa_helm/tasks/message_queue1/assets/index.html

```diff
@@ -0,0 +1,38 @@
+<html>
+
+<head>
+    <link rel="stylesheet" href="/{{ .Release.Namespace }}/{{ .Release.Name }}/info/style.css">
+</head>
+<title>The house of apps: publish a message to a message queue <i>{{ .Release.Namespace }}-{{ .Release.Name }}</i></title>
+
+<h1>publish a message to a message queue</h1>
+<p>
+<ol>
+    <li>
+        Implement an app that <a href="https://www.rabbitmq.com/tutorials" target="_blank">connects to a Rabbit MQ instance</a> and send a message to a message queue every 30 seconds
```

Comment:
I have kept the requirement with every x seconds, otherwise its not clear to the implementer that the app needs to keep running. Lowered the threshold to 30 seconds to that eventually more messages arrive